__tests__/validator.tests.js

```diff
@@ -0,0 +1,79 @@
+const Validator = require('../lib/Validator');
+
+describe('Validator', () => {
+  let validator;
+
+  describe('all fields required', () => {
+    beforeAll(() => {
+      validator = new Validator('age', {
+        type: Number,
+        required: true
+      });
+    });
+
+    it('returns the field', () => {
+      const cat = {
+        name: 'whiskey',
+        age: 18,
+        color: 'grey'
+      };
+
+      expect(validator.validate(cat)).toEqual(18);
+    });
+    
+    it('returns the field cast to type', () => {
+      const cat = {
+        name: 'whiskey',
+        age: '18',
+        color: 'grey'
+      };
+
+      expect(validator.validate(cat)).toEqual(18);
+    });
+    it('returns the field', () => {
```

Comment:
you have two tests (lines 14-19 and 33-40) that say 'it returns the field'-- I'd love it if this was a bit more specific to each test.